_data/team-members.yml

```
@@ -45,7 +45,7 @@ members:
     active: true
   - member: julien
     name: Julien Richard-Foy
-    title: Software Development
+    title: Education Direction 
```

Should this be:
```suggestion
    title: Education Director
```
to match the Academic, Executive and Technical Directors' titles?